### DISCUSSION TITLE
How to change animate_bool time?

### DISCUSSION THREAD

**Discussion by BctfN0HUK7Yg** [emilk/egui#984] [Q&A]
I make animation with animate_bool. But it is very fast. How to make it slowly?

**Comment by BctfN0HUK7Yg** [comment]
when I use special window for changing animation time it's works.
`egui::Window::new("🔧 Settings")  
            .vscroll(true)
            .show(ctx, |ui| {
                ctx.settings_ui(ui);
            });`

but `ui.style_mut().animation_time=0.5; ` doesn't work

**Comment by BctfN0HUK7Yg** [comment]
I don't know how to make Pull Request. 
I change animated_bool function to control each animation time.
`    pub fn animate_bool_with_time(&self, id: Id, value: bool,time:f32) -> f32 {
        let animation_time = time;
        let animated_value =
            self.animation_manager
                .lock()
                .animate_bool(&self.input, animation_time, id, value);
        let animation_in_progress = 0.0 < animated_value && animated_value < 1.0;
        if animation_in_progress {
            self.request_repaint();
        }
        animated_value
    }`

**Comment by emilk** [comment]
Added in https://github.com/emilk/egui/commit/00f965de87ddb07805505a0b7ed3738e0da51a48
